src/frontend/sharedComponents/MapView.js

```diff
@@ -190,14 +201,13 @@ class MapView extends React.Component<Props, State> {
     }
   }
 
-  handleObservationPress = (e: {
-    nativeEvent?: {
-      payload?: {
-        properties?: { id: string },
-      },
-    },
+  handleObservationPress = (event: {
```

Comment:
I guess this has probably been broken for a while - must have changed in a version of mapbox but we never tested this.

Reply:
I think we need to remove the `onPress` handler on the mapview (line 311 I think). We have the onPress handler on the source layer, so we don't want this to be called twice.

Reply:
> I guess this has probably been broken for a while - must have changed in a version of mapbox but we never tested this.

the previous code wasn't really broken. it was just using a now-deprecated API that was introduced in `8.0.0`. Figured it would just make sense to update the implementation to get rid of the deprecation notices

> I think we need to remove the onPress handler on the mapview (line 311 I think). We have the onPress handler on the source layer, so we don't want this to be called twice.

Yeah we can safely remove the handler on the map view. fwiw looks like it doesn't get called twice even if both of them have the handler

Reply:
Done in https://github.com/digidem/mapeo-mobile/pull/867/commits/e52f7dab9b71e6215b5f564cb46627b985c58930